website/database.py

```diff
@@ -663,6 +673,21 @@ def update_class_data(self, id, class_data):
         """Updates a class."""
         CLASSES.update({"id": id}, class_data)
 
+    def store_survey(self, survey):
+        SURVEYS.create(survey)
+
+    def get_survey(self, id):
+        return SURVEYS.get({"id": id})
+
+    def add_survey_responses(self, id, responses):
+        SURVEYS.update({"id": id}, {"responses":  responses})
+
+    def add_skip_survey(self, id):
+        SURVEYS.update({"id": id}, {"skip": True})
+
+    def add_remind_later_survey(self, id):
+        SURVEYS.update({"id": id}, {"skip": date.today().isoformat()})
```

Comment:
I've made skipping the survey generic. This survey will be shown again tomorrow, such as checked in `class_survey in for_teachers.py`. And the text on the button is also loaded there, which now says, 'Remind me tomorrow'. So if we want a survey in the future that is only shown once per week. It would be specified in for example `class_survey in for_teachers.py`. Is that fine?

Reply:
Wonderful!